### ISSUE TITLE
Adding &fields parameter results in Internal Server Error 500

### ISSUE THREAD

**Issue by zak-work** [pushshift/api#7]
It was working few weeks back and today when I clicked it. It is giving me 500
Link: https://api.pushshift.io/reddit/submission/search?q=halloween&size=5&fields=author,full_link

It works without it but I was going for limiting data size being transferred.

**Comment by PaulBGD** [comment]
Yeah this occurred to me too, as well as the metadata object not being returned!

**Comment by odgalvin** [comment]
Same for me, I was about to create an issue when I saw this.

I get "Internal Server Error" whenever I try to use the &fields parameter with submissions, works with comment search though.

**Comment by hkyq** [comment]
Has there been an update on this?
